### PR TITLE
rm back-link from event details page.

### PR DESCRIPTION
**Pull request by stopfstedt**
this came up as unnecessary and potentially confusing during a a11y audit. begone!

fixes https://github.com/ilios/ilios/issues/4774

### CODE REVIEW

**Comment by stopfstedt** [comment]
i'm slapping a `do not merge` label on this since the companion PR https://github.com/ilios/lti-dashboard/pull/1373 is currenly failing.

**Review by jrjohnson — APPROVED** [review]

**Comment by dartajax** [comment]
`DO NOT MERGE` label attached to this ticket - will merge once removed

File: addon/controllers/events.js
Comment (jrjohnson) [inline addon/controllers/events.js:6]: I *think* the way to do this has changed to [getConfig](https://github.com/embroider-build/embroider/tree/main/packages/macros#getownconfig-getconfig-and-getglobalconfig), but I haven't messed with it a lot. I'm ok with this solution, it works for sure, but we're already importing and using these macros other places so it's worth investigating.

**Review by jrjohnson — CHANGES_REQUESTED** [review]

**Review by jrjohnson — APPROVED** [review]

**Comment by stopfstedt** [comment]
i'm starting over on this.

**Pull request closed without merging**
